web/rootfs/etc/cont-init.d/10-config

```diff
@@ -132,6 +132,14 @@ if [[ ! -f /config/config.js ]]; then
         -e "s#// resolution:.*#resolution: 480,#" \
         /config/config.js
 
+    sed -i \
+        "s/stun:meet-jit-si-turnrelay.jitsi.net:443/${JITSI_WEB_STUN_SERVER}/g" \
+        /config/config.js
+
+    sed -i \
+        "/s/enabled: true/enabled: false/g" \
```

Comment:
Also here, please rewrite it to match only one line, not globally. I saw that there are two matches for "enabled: true" in config.js. Can you rewrite it so that only p2p.enabled will be affected and localRecording.enabled will be skipped.